apps/express/package.json

```diff
@@ -2,7 +2,7 @@
     "name": "@csv-parser/express",
     "version": "1.0.0",
     "private": true,
-    "type": "module",
+    "type": "commonjs",
```

Comment:
Is there a reason for setting the type to `commonjs`?
Reverting this change would require to update one import only and would be compliant to all other packages, which use type `module`.